modules/gnome/theme.nix

```diff
@@ -1,4 +1,4 @@
-{ pkgs, config, ... }:
+{ config, inputs, pkgs, ... }:
```

Comment:
I'm not sure `inputs` is defined here. The existing flake inputs are loaded via `stylix/templates.nix`

Reply:
`stylix/templates.nix` is used in the following contexts to extract certain attributes from `inputs`:

- https://github.com/danth/stylix/blob/a9e3ce064a778b386fb88fb152c02ae95aa2cbd2/stylix/darwin/default.nix#L14
- https://github.com/danth/stylix/blob/a9e3ce064a778b386fb88fb152c02ae95aa2cbd2/stylix/hm/default.nix#L15
- https://github.com/danth/stylix/blob/a9e3ce064a778b386fb88fb152c02ae95aa2cbd2/stylix/nixos/default.nix#L16

`stylix/templates.nix` contains the following:

https://github.com/danth/stylix/blob/a9e3ce064a778b386fb88fb152c02ae95aa2cbd2/stylix/templates.nix#L1-L15

To reduce the maintenance burden of manually adding every `inputs` entry to `stylix/templates.nix`, I suggest propagating the entire `inputs` set. In this case, the `stylix/templates.nix` file is obsolete and we replace the propagated `base-*` inputs to `inputs.base-*`.

Should we do that in this PR or a followup PR?

Reply:
I agree, I don't see any reason to maintain a separate list.

Having it in `config.lib.stylix.templates` also means it gets exported, so we could end up with external code relying on our flake inputs.

Splitting into a followup PR would be better, but we'd need this PR to be in a state where everything is functional.

Reply:
Related: https://github.com/danth/stylix/pull/224/commits/dfdc18d8bdce94a4925d8aa3bf78c9c5a79de860

Reply:
> I'm not sure `inputs` is defined here. The existing flake inputs are loaded via `stylix/templates.nix`

https://github.com/danth/stylix/pull/224/commits/dfdc18d8bdce94a4925d8aa3bf78c9c5a79de860 adds the `gnome-shell` input.